json-server/server.js

```diff
@@ -120,6 +120,10 @@ server.post('/api/2.0/dojos/users', (req, res) => {
   }
 });
 
+server.post('/api/2.0/dojos/request-user-invite', (req, res) => {
```

Comment:
You could just add this path to `nonResourcePostUrls` instead. It does the same thing.

Reply:
Will do this :)